library/src/scala/tasty/util/ShowSourceCode.scala

```diff
@@ -859,27 +888,15 @@ class ShowSourceCode[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty
           case _ => this
         }
 
+      case Type.MethodType(paramNames, params, body) =>
+        this += "("
+        printMethodicTypeParams(paramNames, params)
+        this += ") => "
+        printTypeOrBound(body)
+
```

Comment:
Why is this necessary? I assume method types are handled already in refinement case.

Reply:
As far as I can see we do not need it. I will remove it.